Sources/PackageLoading/ManifestJSONParser.swift

```diff
@@ -362,6 +362,8 @@ enum ManifestJSONParser {
                 return .init(rule: .process(localization: localization), path: path.pathString)
             case "copy":
                 return .init(rule: .copy, path: path.pathString)
+            case "embed":
```

Comment:
Should this name be more specific, e.g. `embedInCode`?  Some platforms, like Darwin (with separate Mach-O sections) and Windows (with embedded resources) also support embedding resources inside an executable or a dynamic library.  Since this is supposed to express general intent that can be implemented differently on different platforms, I guess I'm concerned that "embed" is a very general word for a very specifically defined behaviour of always generating inline code.

Reply:
Yah, that sounds reasonable.